::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 26 Nov 2017 17:55:13 +0100
Subject: [PATCH] linux.system.repo: Drop arch for glusterfs repo

Extending the arch list to 'amd64,arm64' would lead to doubling repo
metadata downloads on all archs, while dropping it will allow APT to
only download the repo metadata for the current system arch.

To keep old behavior for amd64 and prevent multiple downloads, let's
just drop the 'architectures' field.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/repo/glusterfs.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/linux/system/repo/glusterfs.yml b/linux/system/repo/glusterfs.yml
index 71d063ac..9afe553c 100644
--- a/linux/system/repo/glusterfs.yml
+++ b/linux/system/repo/glusterfs.yml
@@ -6,7 +6,6 @@ parameters:
       repo:
         glusterfs-ppa:
           source: "deb http://ppa.launchpad.net/gluster/glusterfs-${_param:glusterfs_version}/ubuntu ${_param:linux_system_codename} main"
-          architectures: amd64
           key_id: 3FE869A9
           key_server: keyserver.ubuntu.com
           pin:
